test/e2e/network/service.go

```diff
@@ -2667,6 +2677,15 @@ func createPausePodDeployment(cs clientset.Interface, name, ns string, replicas
 	return deployment
 }
 
+func createPodWithArgs(f *framework.Framework, ns, name string, labels map[string]string, containerPorts []v1.ContainerPort, args ...string) *v1.Pod {
```

Comment:
so, the reachability methods test expects that pods have BOTH ports, so we need to create each pod with 2 containers, and each container binding one of the ports